src/kOS.Safe/Exceptions/KOSDeprecationException.cs

```diff
@@ -6,7 +6,7 @@ public class KOSDeprecationException : KOSException
     {
         protected static string TerseMessageFmt = "As of kOS {0}, {1} is obsolete and has been replaced with {2}";
 
-        public KOSDeprecationException(string version, string oldUsage,string newUsage, string url) :
+        public KOSDeprecationException(string version, string oldUsage,string newUsage) :
```

Comment:
Sorry for being harsh about it.  I just really want that feature to work some day (the help URL).  I know it's just a stub at the moment that doesn't do much yet.

Also, the consequences of this change, percolating through the rest of the code, were bloating the "files changed" count, and I was trying to narrow it down to which files I *really* needed to be looking at to see what changed for subdirectories.